shared/src/main/scala/org/adridadou/openlaw/parser/template/variableTypes/PeriodType.scala

```diff
@@ -55,6 +65,7 @@ case object PeriodType extends VariableType("Period") {
       ( if( period.months > 0 ) s"${period.months}" + " months " else "") +
       ( if( period.weeks > 0 ) s"${period.weeks}" + " weeks " else "") +
       ( if( period.days > 0 ) s"${period.days}" + " days " else "") +
+      ( if( period.hours > 0 ) s"${period.hours}" + " hours " else "") +
```

Comment:
Small bug caught here - internalFormat failed to store hours. Not sure if this somehow affects us retroactively